test/NodeGuard.Tests/Rpc/AuthInterceptorTests.cs

```diff
@@ -0,0 +1,67 @@
+
+using Grpc.Core;
+using NodeGuard.Data.Models;
+using NodeGuard.Data.Repositories.Interfaces;
+
+namespace NodeGuard.Rpc;
+
+public class AuthInterceptorTests
```

Comment:
Can you add Arrange, act, assert comment sections? We've been using them across the whole tests ;)

Reply:
fixeeeed